tests/test_config_gsobject.py

```diff
@@ -515,13 +517,165 @@ def test_devaucouleurs():
+                   'half_light_radius' : 1, 'flux' : 50,
+                   'gsparams' : { 'minimum_fft_size' : 256 }
+                 },
+        'gal6' : { 'type' : 'InclinedExponential' , 'inclination' : 0.7 * galsim.radians,
+                   'half_light_radius' : 1, 'flux' : 50,
+                   'gsparams' : { 'maximum_fft_size' : 64 }
+                 },
+    }
+
+    gal1a = galsim.config.BuildGSObject(config, 'gal1')[0]
+    gal1b = galsim.InclinedExponential(inclination=0.1 * galsim.radians, half_light_radius=2)
+    gsobject_compare(gal1a, gal1b)
+
+    gal2a = galsim.config.BuildGSObject(config, 'gal2')[0]
+    gal2b = galsim.InclinedExponential(inclination=21 * galsim.degrees, scale_radius=0.7, flux=100)
+    gsobject_compare(gal2a, gal2b)
+
+    gal3a = galsim.config.BuildGSObject(config, 'gal3')[0]
+    gal3b = galsim.InclinedExponential(inclination=0.3 * galsim.radians, scale_radius=0.35,
+                                  scale_height=0.23, flux=1.e6)
+    gal3b = gal3b.shear(q=0.6, beta=0.39 * galsim.radians)
+    gsobject_compare(gal3a, gal3b)
+
+    gal4a = galsim.config.BuildGSObject(config, 'gal4')[0]
+    gal4b = galsim.InclinedExponential(inclination=0.7 * galsim.radians, half_light_radius=1,
+                                  scale_h_over_r=0.2, flux=50)
+    gal4b = gal4b.dilate(3).shear(e1=0.3).rotate(12 * galsim.degrees)
+    gal4b = gal4b.lens(0.03, -0.05, 1.03).shift(dx=0.7, dy=-1.2)
+    gsobject_compare(gal4a, gal4b)
+
+    gal5a = galsim.config.BuildGSObject(config, 'gal5')[0]
+    gsparams = galsim.GSParams(minimum_fft_size=256)
+    gal5b = galsim.InclinedExponential(inclination=0.7 * galsim.radians, half_light_radius=1, flux=50, gsparams=gsparams)
+    gsobject_compare(gal5a, gal5b, conv=galsim.Gaussian(sigma=1))
+
+    try:
+        # Make sure they don't match when using the default GSParams
+        gal5c = galsim.InclinedExponential(inclination=0.7 * galsim.radians, half_light_radius=1, flux=50)
+        np.testing.assert_raises(AssertionError, gsobject_compare, gal5a, gal5c,
+                                 conv=galsim.Gaussian(sigma=1))
+
+        # For the maximum_fft_size test, we need to do things a little differently
+        # We lower maximum_fft_size below the size that SBProfile wants this to be,
+        # and we check to make sure an exception is thrown.  Of course, this isn't how you
+        # would normally use maximum_fft_size.  Normally, you would raise it when the default
+        # is too small.  But to construct the test that way would require a lot of memory
+        # and would be rather slow.
+        gal6a = galsim.config.BuildGSObject(config, 'gal6')[0]
+        gal6b = galsim.InclinedExponential(inclination=0.7 * galsim.radians, half_light_radius=1, flux=50)
+        np.testing.assert_raises(RuntimeError, gsobject_compare, gal6a, gal6b,
+                                 conv=galsim.Gaussian(sigma=1))
```

Comment:
The above gal5c test is sufficient for this.  You don't need to also do the maximum_fft_size test as well.  We did that for Sersic to make sure that *something* tested maximum_fft_size.  But here, we only need to test that the GSParams option is being parsed.  And gal5c tests that already.